{
  "Class": "vulnerability",
  "ID": "oval:com.redhat.unaffected:def:201920478",
  "Version": "631",
  "Metadata": {
    "Title": "Unaffected components for: CVE-2019-20478 python-ruamel-yaml: code execution through load() method with an untrusted argument (moderate)",
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2019-20478",
        "RefURL": "https://access.redhat.com/security/cve/CVE-2019-20478"
      }
    ],
    "Description": "Red Hat's versions of the associated software have been determined to NOT be affected by CVE-2019-20478.",
    "Advisory": {
      "From": "secalert@redhat.com",
      "Severity": "Moderate",
      "Issued": {},
      "Updated": {}
    }
  },
  "Criteria": {
    "Operator": "OR",
    "Criterions": [
      {
        "Comment": "Red Hat Enterprise Linux must be installed",
        "TestRef": "oval:com.redhat.cve:tst:201610089010"
      }
    ],
    "Criterias": [
      {
        "Operator": "AND",
        "Criterions": [
          {
            "Comment": "Red Hat Enterprise Linux 6 is installed",
            "TestRef": "oval:com.redhat.cve:tst:201610089009"
          },
          {
            "Comment": "python-ruamel-yaml is installed",
            "TestRef": "oval:com.redhat.unaffected:tst:201920478001"
          },
          {
            "Comment": "python-ruamel-yaml is not installed",
            "TestRef": "oval:com.redhat.unaffected:tst:201920478002"
          }
        ]
      }
    ]
  }
}